gdk/commands/component/transformer/BuildRecipeTransformer.py

```diff
@@ -32,6 +34,15 @@ def transform(self, build_folders):
 
         component_recipe = CaseInsensitiveRecipeFile().read(self.project_config.recipe_file)
         self.update_component_recipe_file(component_recipe, build_folders)
+
+        logging.info("Validating the recipe against the Greengrass recipe schema.")
+        try:
+            recipe_schema_path = utils.get_static_file_path(consts.recipe_schema_file)
+            validator = RecipeValidator(recipe_schema_path)
```

Comment:
This can also raise `jsonschema.exceptions.SchemaError` where the input schema does not abide by jsonschema standards. This should be accounted for in case the local recipe schema was touched accidentally. Will also be useful in the future to track if the schema we pull down had some unexpected issues.

Reply:
Added handling for this and manually tested with an invalid schema

Reply:
This more of a CLI/service error. It should be logged differently like this one here: https://github.com/aws-greengrass/aws-greengrass-gdk-cli/blob/main/gdk/CLIParser.py#L198-L199. 

Also, add an integ test, just to validate the schema like we've for other schema here: https://github.com/aws-greengrass/aws-greengrass-gdk-cli/blob/main/integration_tests/gdk/common/test_integ_model_actions.py#L12

Reply:
Updated in latest commit. The error message now directs to the github issues page and added an additional integration test to validate the recipe schema.